testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestTextFileHiveTable.java

```diff
@@ -132,7 +132,33 @@ public void testInsertTextFileSkipHeaderFooter()
                 "STORED AS TEXTFILE " +
                 "TBLPROPERTIES ('skip.header.line.count'='1', 'skip.footer.line.count'='1')");
         assertThatThrownBy(() -> onTrino().executeQuery("INSERT INTO test_textfile_skip_header_footer VALUES (1)"))
-                .hasMessageMatching(".* Inserting into Hive table with skip.header.line.count property not supported");
+                .hasMessageMatching(".* Inserting into Hive table with skip.footer.line.count property not supported");
         onHive().executeQuery("DROP TABLE test_textfile_skip_header_footer");
     }
+
+    @Test
+    public void testCreateTextFileTableAsSelectSkipHeaderFooter()
+    {
+        onHive().executeQuery("DROP TABLE IF EXISTS test_create_textfile_skip_header");
+        onTrino().executeQuery(
+                "CREATE TABLE test_create_textfile_skip_header " +
+                        "WITH ( " +
+                        "   format = 'TEXTFILE', " +
+                        "   skip_header_line_count = 1 " +
+                        ") " +
+                        "AS SELECT 1  AS col_header;");
+        assertThat(query("SELECT * FROM test_create_textfile_skip_header")).containsOnly(row(1));
```

Comment:
please validate actual contents.
Also query from HIve and see if it also can read expected data after Trino CTAS

Reply:
ok